localstorage/firebase-worker.js

```diff
@@ -46,12 +56,21 @@ async function initializeApp(withPersistence) {
   if (withPersistence) {
     console.log('Starting Firestore with persistence.');
     await self.db.enablePersistence({experimentalForce: true}).then(() => {
-      postMessage('takePersistence');
+      postMessage('persistenceTaken');
     });
   } else {
     console.log(
       'Persistence unavailable. Starting Firestore with persistence will error.'
     );
     // await self.db.enablePersistence();  // This will error
-  } 
+  }
+
+  // Listen to data.
+  self.db.collection('followers')
+    .onSnapshot({ includeMetadataChanges: true }, function(snapshot) {
+      snapshot.docChanges().forEach(function(change) {
```

Comment:
Would it be simpler / more straightforward to just add a document listener on /followers/user1 directly?